charts/cosigned/templates/webhook/cosign_secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    {{- include "cosigned.labels" . | nindent 4 }}
+  name: {{ template "cosigned.fullname" . }}-cosign-key
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  cosign.key: {{ .Values.cosign.cosignKey}}
```

Comment:
`{{ required "A cosign key is required" .Values.cosign.cosignKey }}`
> see
https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function

This helps checking for values that are required.

An other idea would could be to add a `values.schema.json` => https://helm.sh/docs/topics/charts/#schema-files

Reply:
thanks for that, i forgot to add that we used to have it but I lost in the transition, will update